### PR TITLE
Support for creating a Queue with options

### PR DESCRIPTION
**Pull request by kody-abe**
Per the spec https://github.com/OptimalBits/bull/blob/master/REFERENCE.md#queue creating a Queue can support additional options. This adds support for that.

### CODE REVIEW

**Comment by erikengervall** [comment]
> Per the spec https://github.com/OptimalBits/bull/blob/master/REFERENCE.md#queue creating a Queue can support additional options. This adds support for that.

Hey @kody-abe!

Thanks for your PR. I think this is a great addition but would love to see it reflected in the project's documentation :)

File: index.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ module.exports = {
   UI: UI(),
   createQueues: redis => {
     return {
-      add: name => {
-        const queue = new Queue(name, redis)
+      add: (name, opts) => {
```
Comment (erikengervall) [inline index.js]: Please reflect these changes in the project's README :)

**Comment by vcapretz** [comment]
Thanks for the PR! 
Would be nice the update on the README page, I'll merge after it!

**Comment by kody-abe** [comment]
@vcapretz Added, please let me know if you wanted something more specific or called out

File: README.md [hunk 2]
```diff
@@ -47,6 +47,9 @@ And then you can setup how your queues will work:
 
 ```js
 const helloQueue = queues.add('helloQueue') // adds a queue
+const helloQueueWithOpts = queues.add('helloQueue', {
+    prefix: 'hello',
+}) // adds a queue as QueueOptions https://github.com/OptimalBits/bull/blob/master/REFERENCE.md#queue
```
Comment (vcapretz) [inline README.md]:
```suggestion
}) // adds a queue with QueueOptions https://github.com/OptimalBits/bull/blob/master/REFERENCE.md#queue
```

**Comment by vcapretz** [comment]
@kody-abe it's perfect for now! we will eventually have more docs and use cases! 

thanks!

**Pull request merged**